### PR TITLE
Add mintso config parameters

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What is changed, added or deleted? (Required)

Adds TiFlash MinTSO config parameters

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/18621, https://github.com/pingcap/docs-cn/pull/18542

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -250,6 +250,15 @@ delta_index_cache_size = 0
     ## 从 v7.4.0 引入，表示是否开启 TiFlash 资源管控功能。当设置为 true 时，TiFlash 会使用 Pipeline Model 执行模型。
     enable_resource_control = true
 
+    ## 从 v6.0.0 引入，表示一个 resource group 中最多可使用的线程数量，默认值为 5000
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:253]:
```suggestion
    ## 从 v6.0.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示一个 resource group 中最多可使用的线程数量，默认值为 5000
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -250,6 +250,15 @@ delta_index_cache_size = 0
...
+    task_scheduler_thread_soft_limit = 5000
+
+    ## 从 v6.0.0 引入，表示全局最多可使用的线程数量，默认值为 10000
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:256]:
```suggestion
    ## 从 v6.0.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示全局最多可使用的线程数量，默认值为 10000
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -250,6 +250,15 @@ delta_index_cache_size = 0
...
+    task_scheduler_thread_hard_limit = 10000
+
+    ## 从 v6.4.0 引入，表示 TiFlash 实例同时运行的 query 数量，默认值为 0，即两倍 vcpu 数量
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:259]:
```suggestion
    ## 从 v6.4.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示 TiFlash 实例最多同时运行的 query 数量，默认值为 0，即两倍 vcpu 数量
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -250,6 +250,15 @@ delta_index_cache_size = 0
...
     enable_resource_control = true
 
+    ## 从 v6.0.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示一个 resource group 中最多可使用的线程数量，默认值为 5000
+    task_scheduler_thread_soft_limit = 5000
+
+    ## 从 v6.0.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示全局最多可使用的线程数量，默认值为 10000
+    task_scheduler_thread_hard_limit = 10000
+
+    ## 从 v6.4.0 引入，用于 [MinTSO 调度器](/tiflash/tiflash-mintso-scheduler.md)，表示 TiFlash 实例最多同时运行的 query 数量，默认值为 0，即两倍 vcpu 数量
+    task_scheduler_active_set_soft_limit = 0
```
[the remark this answers is not in the document]
Comment (xzhangxian1008) [reply tiflash/tiflash-configuration.md]: > 代码块里需要用完整的链接，先替换一下，后续计划把 TiFlash 的每个配置参数调成文内小标题的格式。

done

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -250,6 +250,15 @@ delta_index_cache_size = 0
     ## 从 v7.4.0 引入，表示是否开启 TiFlash 资源管控功能。当设置为 true 时，TiFlash 会使用 Pipeline Model 执行模型。
     enable_resource_control = true
 
+    ## 从 v6.0.0 引入，用于 MinTSO 调度器，表示一个资源组中最多可使用的线程数量，默认值为 5000。关于 MinTSO 调度器，详见 https://docs.pingcap.com/zh/tidb/dev/tiflash-mintso-scheduler
+    task_scheduler_thread_soft_limit = 5000
+
+    ## 从 v6.0.0 引入，用于 MinTSO 调度器，表示全局最多可使用的线程数量，默认值为 10000。关于 MinTSO 调度器，详见 https://docs.pingcap.com/zh/tidb/dev/tiflash-mintso-scheduler
+    task_scheduler_thread_hard_limit = 10000
+
+    ## 从 v6.4.0 引入，用于 MinTSO 调度器，表示 TiFlash 实例最多可同时运行的查询数量，默认值为 0，即两倍的 vCPU 数量。关于 MinTSO 调度器，详见 https://docs.pingcap.com/zh/tidb/dev/tiflash-mintso-scheduler
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:259]:
```suggestion
    ## 从 v6.4.0 引入，用于 MinTSO 调度器，表示一个 TiFlash 实例中最多可同时运行的查询数量，默认值为 0，即两倍的 vCPU 数量。关于 MinTSO 调度器，详见 https://docs.pingcap.com/zh/tidb/dev/tiflash-mintso-scheduler
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by windtalker — APPROVED** [review]
lgtm

**Pull request merged**